src/array_api_typing/_namespace.py

```diff
@@ -0,0 +1,27 @@
+"""Static typing support for the array API standard."""
+
+__all__ = ["HasNamespace"]
+
+from typing import Protocol, runtime_checkable
+
+
+@runtime_checkable
```

Comment:
Personally I'm not a big fan of `@runtime_checkable`, mostly because of all the false positives it can give, e.g. 

```py
>>> isinstance(type(x), xpt.HasNamespace)
True
```

Reply:
How the hell does that pass?

Reply:
That's not great.

Reply:
Removed the runtime-checkable. We can add that in later if needed.